gui.js

```diff
@@ -4472,22 +4511,6 @@ IDE_Morph.prototype.getURL = function (url) {
     }
 };
 
-IDE_Morph.prototype.getURLsbeOrRelative = function (url) {
```

Comment:
I hope he doesn't mind! After we did the whole `baseURL` thing is was no longer loading from s.b.e so it wasn't even doing it's job. (Also technically, though there was a big argument in July 2014 about this, the fallback mechanism part wasn't working reliably.)

However, in my refactoring, I've realized that we don't need the `baseURL` anyway... We just want to load everything relative to the `snap.html` file anyway, right? 
That said, if we do need it, I now only need to change 1 line of code, not 5. ;-)